include/micm/solver/lu_decomposition.inl

```diff
@@ -264,52 +278,58 @@ namespace micm
       auto lkj_uji = lkj_uji_.begin();
       auto uii = uii_.begin();
       is_singular = false;
-      const std::size_t n_cells = std::min(A.GroupVectorSize(), A.size() - i_group * A.GroupVectorSize());
+      std::size_t n_cells = std::min(A_GroupVectorSize, A_Size - i_group * A_GroupVectorSize);
```

Comment:
Is there a reason this has to be non-`const` now?  I might be missing a detail below but nothing sticks out as modifying it.

Reply:
No, It doesn't. I put `const` back in, thanks for noticing!